assets/js/modules/pagespeed-insights/components/common/FieldReportMetrics.js

```diff
@@ -118,9 +117,6 @@ export default function FieldReportMetrics( { data, error } ) {
 					/>
 				</tbody>
 			</table>
-			<div className="googlesitekit-pagespeed-report__row googlesitekit-pagespeed-report__row--single-item googlesitekit-pagespeed-report__row--last">
```

Comment:
The `googlesitekit-pagespeed-report__row--last` class needs to move up to the row for "Cumulative Layout Shift" as it is on the Lab Data tab to prevent a layout shift when toggling between tabs 😄 

See here https://google.github.io/site-kit-wp/storybook/pull/2481/?path=/story/pagespeed-insights-module-components--dashboard-widget

Reply:
@aaemnnosttv Can you clarify, I'm not sure what you mean. The only layout shift I see between the tabs is due to the descriptions for Lab vs Field data being of different lengths.

Reply:
Clarified back-channel 😄